aws-logs-loggroup/src/test/java/software/amazon/logs/loggroup/CreateHandlerTest.java

```diff
@@ -137,12 +139,19 @@ public void handleRequest_SuccessGeneratedLogGroupName() {
                 .desiredResourceState(model)
                 .build();
 
+        final Map<String, String> systemTags = new HashMap<>();
+        systemTags.put("aws:cloudformation:stack-name", "unit_test_Stack");
+        request.setSystemTags(systemTags);
+        request.setClientRequestToken("4b90a7e4-b790-456b-a937-0cfdfa212fed");
+        request.setLogicalResourceIdentifier("taskDefinition");
+
         final ProgressEvent<ResourceModel, CallbackContext> response = handler.handleRequest(proxy, request, null, logger);
 
         assertThat(response).isNotNull();
         assertThat(response.getStatus()).isEqualTo(OperationStatus.SUCCESS);
         assertThat(response.getCallbackContext()).isNull();
         assertThat(response.getCallbackDelaySeconds()).isEqualTo(0);
+        assertThat(response.getResourceModel().getLogGroupName()).contains("unit_test_Stack");
```

Comment:
Better to use `startsWith` to verify that the stack name  appended first.

Reply:
- is that a requirement ?
- if any logic doesn't depend on it I would prefer contains

Reply:
Not a requirement, but it's closing to the implementation. Unless we change our implementation, the expected name should start with stack name.  

And actually start with stack name is using by some as they can use it in some cases such IAM policy for resource level permission control.

Reply:
sounds good

Reply:
done